pallets/nfts/src/weights.rs

```diff
@@ -1016,14 +1022,14 @@ impl WeightInfo for () {
 	/// Proof: `Nfts::PendingSwapOf` (`max_values`: None, `max_size`: Some(71), added: 2546, mode: `MaxEncodedLen`)
 	fn burn() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `584`
+		//  Measured:  `634`
 		//  Estimated: `4326`
-		// Minimum execution time: 46_000_000 picoseconds.
-		Weight::from_parts(47_000_000, 4326)
+		// Minimum execution time: 40_000_000 picoseconds.
+		Weight::from_parts(44_000_000, 4326)
 			.saturating_add(RocksDbWeight::get().reads(6_u64))
 			.saturating_add(RocksDbWeight::get().writes(8_u64))
 	}
-	/// Storage: `Nfts::Collection` (r:1 w:0)
+/// Storage: `Nfts::Collection` (r:1 w:0)
```

Comment:
Weird - makes it seem like something has been copy pasted?

Reply:
Oh yes, I only run benchmarking on related methods and replace them with the main `weight.rs` file.